pkgs/by-name/zi/zizmor/package.nix

```diff
@@ -1,29 +1,32 @@
 {
   lib,
   fetchFromGitHub,
+  nix-update-script,
   rustPlatform,
   testers,
   zizmor,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "zizmor";
-  version = "1.4.1";
+  version = "1.5.1";
 
   src = fetchFromGitHub {
     owner = "woodruffw";
     repo = "zizmor";
     tag = "v${version}";
-    hash = "sha256-cOfFfij/w0W4FcJg6KnIQxbSgC+cp0iyZqNPSbKheoQ=";
+    hash = "sha256-G0cayE8UDahLYZl2HG8315YYUOhH5u6L/VRWHqqavPk=";
   };
 
   useFetchCargoVendor = true;
-  cargoHash = "sha256-XXV3aOIbzIZ8B7UlkhSeiHoa9H9GJiF8ZMzLS0e0R0w=";
+  cargoHash = "sha256-CRjr4X525t7xaFsnX3gOy4+HJis5T8nw6zrhkW60Bpw=";
 
   passthru.tests.version = testers.testVersion {
     package = zizmor;
   };
```

Comment:
Lets change this to `versionCheckHook` while we're at it

Reply:
I pushed the change.